### ISSUE TITLE
Can't find local PC over ethernet, only over Wi-Fi (all on the same network)

### ISSUE THREAD

**Issue by King-Cramhole** [moonlight-stream/moonlight-ios#594]
Hi!

I've gone through all the steps for troubleshooting however, I am still not able to connect to my Win 11 desktop PC via ethernet through Moonlight. Adding my IP manually also does not find it.

Been trying to setup Moonlight streaming from my Steam Deck to my Windows desktop PC (tried using Sunshine and GFE). However, my desktop PC refuses to show up in Moonlight no matter what I try. I've tried all suggested steps on Github as well as ton I found on Google and Reddit. Manually entering my IP also does nothing. I've reinstalled all apps and rebooted hardware numerous times. I've run the Internet hosting tool and it ran successful. All ports have been added to my routers port forwarding. 

After giving up I installed Moonlight on my Android phone for fun just to see what would happen. To my surprise it located my PC over WiFi instantly. So then I connected my PC to WiFi instead of ethernet and then my Steam Deck displayed my local PC right away. Would anyone know why it's not finding my PC over a direct Ethernet connection on the same network? Even disabling my firewall didn't make a difference.

Someone suggested it could be a "network misconfiguration" though I'm not exactly sure what that could entail. Any suggestions on what might be able to resolve this?

**Comment by H4ZEDAWAHRIOR** [comment]
Did you find an answer to your question? i'm having similar issues and no help so far...

**Comment by King-Cramhole** [comment]
> Did you find an answer to your question? i'm having similar issues and no help so far...

Yes and no. Despite my PC being connected to my router directly through Ethernet, I switched it to WiFi instead and then suddenly my PC was found. Not ideal so it's just a workaround.

**Comment by H4ZEDAWAHRIOR** [comment]
Sooo after fighting a while with this thing. Ive found a solution that works for me. 

I needed to enable "Gamestream" on the "Shield" tab on the Geforce Experience app  on my host/PC. 

It works now. Tell me if it works for you too!

**Comment by King-Cramhole** [comment]
I tried that initially but that didn't work for me :(

**Comment by zx1997s** [comment]
I had the same problem on my iPhone, but switched to an Android phone and got a good connection whether using WiFi or 5G.  I'm pretty sure this is not a coincidence.

**Comment by A4toX** [comment]
you need to expose your computer to the public network. While on the local network, enter the public IP or domain name to establish a connection once. Afterward, switch to Ethernet, and you should be able to connect successfully. This is due to Apple's restrictions

**Comment by Left-Empty** [comment]
In my case, the issue was only happening on external acces, no issue on LAN.
When the Sunshine computer (running Windows 10) is connected on Ethernet, Moonlight cannot find from an external netwrok. When it is connected through Wi-fi, no issue.
I checked the network settings on my PC and routeur, but I could not find spot any difference, as I leave the security quite tied on the Windows machine.
